components/director/internal/open_resource_discovery/service.go

```diff
@@ -1617,3 +1799,35 @@ func addFieldToLogger(ctx context.Context, fieldName, fieldValue string) context
 	logger = logger.WithField(fieldName, fieldValue)
 	return log.ContextWithLogger(ctx, logger)
 }
+
+func createWebhookInput(credentialExchangeStrategyJSON gjson.Result, tenantMappingData CredentialExchangeStrategyTenantMapping) *graphql.WebhookInput {
+	inputMode := graphql.WebhookMode(tenantMappingData.Mode)
+	return &graphql.WebhookInput{
+		URL: str.Ptr(credentialExchangeStrategyJSON.Get(callbackURLProperty).String()),
+		Auth: &graphql.AuthInput{
+			AccessStrategy: str.Ptr(string(accessstrategy.CMPmTLSAccessStrategy)),
```

Comment:
Redundant string conversion

Reply:
Not redundant. IDE says it is, but at runtime, an error is thrown. I also thought that it is redundant but apparently, it is not. :( 
<img width="1375" alt="image" src="https://github.com/kyma-incubator/compass/assets/17293744/799e8fba-62de-44c3-9f14-9a74c79b4a87">